src/libsync/discovery.cpp

```diff
@@ -421,7 +421,7 @@ void ProcessDirectoryJob::processFile(PathTuple path,
     const char *hasLocal = localEntry.isValid() ? "true" : _queryLocal == ParentNotChanged ? "db" : "false";
     const auto serverFileIsLocked = (serverEntry.isValid() ? (serverEntry.locked == SyncFileItem::LockStatus::LockedItem ? "locked" : "not locked")  : "");
     const auto localFileIsLocked = dbEntry._lockstate._locked ? "locked" : "not locked";
-    qCInfo(lcDisco).nospace() << "Processing " << path._original
+    qCDebug(lcDisco).nospace() << "Processing " << path._original
```

Comment:
@mgallien not sure about this one if we disable debug level by default, we use this line often when investigating issues

Reply:
I reverted this change